rust/connlib/clients/android/src/lib.rs

```diff
@@ -224,10 +224,11 @@ impl Callbacks for CallbackHandler {
             .map(Some)
             .map_err(|source| CallbackError::CallMethodFailed { name, source })
         })
+        .expect("onUpdateRoutes callback failed")
```

Comment:
I'm a bit wary about this, in some platforms for ipv4/ipv6-only stack I think this can fail during normal operation.

Reply:
If it can fail during normal operation, the Android side should catch the exception and not just abort.

I want to treat these panics as a bug so I'd rather fix the bug than hide the symptoms.